tests/testthat/test-read_dese_childhood_serves.R

```diff
@@ -0,0 +1,22 @@
+# testing read_dese_childhood_services function
+
+test_that("Read of DESE childhood services Data works", {
+
+  dese <- read_dese_childhood_services()        # fails due to time out. Function needs fixing
```

Comment:
place the function within a `tryCatch` and on failure run it again. Have a look at the docs for tryCatch, then give me a call to discuss

Reply:
Ok, will try that.

Reply:
The function and the test of the function are now working fine. I tried several times. Might have been due to my internet issues? Do you still want me to add a tryCatch clause?

Reply:
yes but perhaps just add the tryCatch to the test. Beau and I have both been having timeout issues so it is worthwhile making the test more robust

Reply:
done.